README.md

```diff
@@ -2,20 +2,20 @@
+| **[Getting Started](Getting_Started.ipynb)**                               | Py       | Intro         | How to connect, some simple data access                                                                                                                                                                                                                                                                                                                                                 | [launch](https://idr-analysis.openmicroscopy.org/vae/user-redirect/Getting_Started.ipynb)                 |
+| **[Figure 1 Sample of Phenotypes](Figure_1_Sampling_of_Phenotypes.ipynb)** | Py       | IDR Paper         | Reproduces Fig. 1 of the paper: downloads annotations from all screens and computes and plots some statistics                                                                                                                                                                                                                                                                           | [launch](https://idr-analysis.openmicroscopy.org/vae/user-redirect/Figure_1_Sampling_of_Phenotypes.ipynb) |
+| **[Gene Network](GeneNetwork.ipynb)**                                      | Py       | IDR Paper         | Reproduces Fig. 2 of the paper: downloads annotations from 3 screens,with a phenotype in common, queries StringDB for interactions and plots the resulting network. Uses a conversion table for orthologues and gene identifiers which was built offline using biomart (see article for more details). It uses [bokeh](http://bokeh.pydata.org/) and [NetworkX](https://networkx.github.io/) to display networks. | [launch](https://idr-analysis.openmicroscopy.org/vae/user-redirect/GeneNetwork.ipynb)                     |
+| **[Condensation Bulk Annotations](CondensationBulkAnnotations.R.ipynb)**   | **R**    | Study details | Reproduces some plots from the original paper showing how one can interact with OMERO from R                                                                                                                                                                                                                                                                                   | [launch](https://idr-analysis.openmicroscopy.org/vae/user-redirect/CondensationBulkAnnotations.R.ipynb)   |
+| **[PCA analysis of Charm features](PCAanalysisOfCharmFeatures.ipynb)**     | Py       | Study details | Shows how to access the computed CHARM features using OMERO.tables,and performs some analysis on them, showing that single cell information can be accessed from generic tile-based features,without segmentation.                                                                                                                                                                      | [launch](https://idr-analysis.openmicroscopy.org/vae/user-redirect/PCAanalysisOfCharmFeatures.ipynb)      |
```

Comment:
space before ``and performs`` and before ``without segmentation.                                                     ``